src/trading/platform_aware.py

```
@@ -0,0 +1,270 @@
+        priority_fee_manager: PriorityFeeManager,
+        slippage: float = 0.25,
+        max_retries: int = 5,
+    ):
+        """Initialize platform-aware token seller."""
+        self.client = client
+        self.wallet = wallet
+        self.priority_fee_manager = priority_fee_manager
+        self.slippage = slippage
+        self.max_retries = max_retries
+
+    async def execute(self, token_info: TokenInfo, *args, **kwargs) -> TradeResult:
```

Ruff (0.12.2)

Missing type annotation for `*args`

(ANN002)